### PR TITLE
Fix color picker and perspectives order #LMR-1833 #LMR-1834

### PR DESCRIPTION
**Pull request by marvec**

### CODE REVIEW

File: src/app/shared/picker/color-picker/color-picker.component.ts [hunk 1]
```diff
@@ -100,13 +105,20 @@ export class ColorPickerComponent implements OnInit, AfterViewInit {
...
         __this.select(color.toHexString());
       },
       move: function(color) {
         __this.preview(color.toHexString());
       },
+      hide: function(color) {
+        if (!__this.savingColor) {
+          __this.colorChange.emit(__this.originalColor);
+        }
+        __this.savingColor = false;
```
Comment (kubedo8) [inline src/app/shared/picker/color-picker/color-picker.component.ts]: I'm not sure if i inderstand correctly, but `__this` refers to component's `this`?

Comment (marvec) [reply src/app/shared/picker/color-picker/color-picker.component.ts]: Yes, this is an old construct threre. I did not change it. Should I? I could probably use bind...

Comment (kubedo8) [reply src/app/shared/picker/color-picker/color-picker.component.ts]: Thanks! Now it's more clear what's going on!

File: src/app/workspace/project/project-settings.component.ts [hunk 2]
```diff
@@ -134,7 +134,7 @@ export class ProjectSettingsComponent implements OnInit {
           filter(project => !!project)
         )
         .subscribe(project => {
-          this.project$.next(project);
+          this.project$.next({...project});
```
Comment (kubedo8) [inline src/app/workspace/project/project-settings.component.ts]: You forgot to include organization, didn't you?

**Pull request merged**